src/http/formdata/generator.cr

```diff
@@ -0,0 +1,118 @@
+module HTTP::FormData
+  # Builds a multipart/form-data message.
+  #
+  # ### Example
+  #
+  # ```
+  # io = IO::Memory.new
+  # builder = HTTP::FormData::Builder.new(io, "aA47")
+  # builder.field("name", "joe")
+  # file = IO::Memory.new "file contents"
+  # builder.file("upload", io, FileMetadata.new(filename: "test.txt"))
+  # builder.finish
+  # io.to_s # => "--aA47\r\nContent-Disposition: form-data; name=\"name\"\r\n\r\njoe\r\n--aA47\r\nContent-Disposition: form-data; name=\"upload\"; filename=\"test.txt\"\r\n\r\nfile contents\r\n--aA47--"
+  # ```
+  class Builder
```

Comment:
Could you rename the file (and the specs) to be named "builder" instead of "generator", now that the types are also named "Builder"?